script/upgrades/dependencies.json

```diff
@@ -60,6 +64,8 @@
     "WatchdogMultisig": "0x823655c966be3b6344Efd4D2A0FE8d0a1e3D691B",
     "MentoFoundationMultisig": "0x707D2B6Bd0acf76821AD5A09A1C6F43d6d6f692c",
     "MentoLiquiditySupport": "0x6bD481a12cb2790E7EE805b9E6e7E91917DeEe6a",
-    "FractalSigner": "0x2fCAb633adFA6aF8266025D63228047033c3ceD0"
+    "FractalSigner": "0x2fCAb633adFA6aF8266025D63228047033c3ceD0",
+    "Chainlink.CELOUSD": "0x022F9dCC73C5Fb43F2b4eF2EF9ad3eDD1D853946",
+    "Chainlink.PHPUSD": "0xfF4998732Cf78f66416d46b3bE8F7068F25Fae11"
```

Comment:
I think the latest one here is `0x995c827326Bc7393BFA28785f98744548874B2f0`? 

That's what I got from running the `UpdateMockChainlinkAggregators` script

Reply:
True, I didn't update it in my last deployment, grr.